cloudflared/config.yaml

```diff
@@ -9,7 +9,6 @@ codenotary: dev@brenner.tech
 init: false
 hassio_api: true
 hassio_role: homeassistant
-watchdog: "http://[HOST]:36500/healthcheck"
```

Comment:
Do you think that we should completely remove watchdog for the add-on?
I think it would be great to have some sort of "auto-restart" if it crashes.
I am fine with removing it for now and thinking of another solution, just wanted to mention it here.

Reply:
I would suggest moving this (as recommended) to Dockerfile as HEALTHCHECK statement, but this needs to be tested first.

The current logic without watchdog statement inside config.yaml will (when Watchdog switch is set to on) just restart the add-on when it crashes with exit code != 0